bears/c_languages/GNUIndentBear.py

```diff
@@ -23,18 +23,125 @@ class GNUIndentBear:
     def create_arguments(filename, file, config_file,
                          max_line_length: int=80,
                          use_spaces: bool=True,
+                         blank_lines_after_declarations: bool=False,
+                         blank_lines_after_procedures: bool=False,
+                         blank_lines_after_commas: bool=False,
+                         ignore_newlines: bool=True,
+                         braces_on_if_line: bool=False,
+                         braces_on_func_def_line: bool=False,
+                         cuddle_else: bool=False,
+                         cuddle_do_while: bool=False,
+                         case_indentation: int=0,
+                         space_special_semicolon: bool=True,
+                         declaration_indent: int=0,
+                         brace_indent: bool = False,
+                         continue_at_parentheses: bool=True,
+                         continuation_indentation: int=0,
+                         gnu_style: bool=False,
+                         k_and_r_style: bool=False,
+                         linux_style: bool=False,
                          tab_width: int=SpacingHelper.DEFAULT_TAB_WIDTH,
                          indent_cli_options: str=''):
         """
-        :param max_line_length:    Maximum number of characters for a line.
-        :param use_spaces:         True if spaces are to be used, else tabs.
-        :param tab_width:          Number of spaces per indent level.
-        :param indent_cli_options: Any command line options the indent binary
-                                   understands. They will be simply passed
-                                   through.
+        :param max_line_length:
+            Maximum number of characters for a line.
+        :param use_spaces:
+            True if spaces are to be used, else tabs.
+        :param blank_lines_after_declarations:
+            Forces blank lines after the declarations.
+        :param blank_lines_after_procedures:
+            Force blank lines after procedure bodies.
+        :param braces_on_if_line:
+            Puts the brace ``{`` on same line with if.
+
+            Example: If ``True``,
+            ```
+            if (x > 0)
+            {
+            ```
+            changes to
+            ```
+            if (x > 0) {
+            ```
+        :param braces_on_func_def_line:
+            Puts the brace `{` on same line with the function declaration.
+        :param cuddle_else:
+            Cuddle else and preceding ``}``.
+
+            Example: If ``True``,
+            ```
+            if (...) {
+                ....
+            }
+            else {
+            ```
+            changes to
+            ```
+            if (...) {
+                ....
+            } else {
+            ```
+
+        :param cuddle_do_while:
```

Comment:
i'd change ``cuddle`` as a term. i understood what it does, but honestly it doesnt fit here ^^